test/TestUtilities/Test.Utility/TestFileSystemUtility.cs

```diff
@@ -36,59 +36,86 @@ private static bool SkipCleanUp()
 
         private static string GetRootDirectory()
         {
-            var repoRoot = GetRepositoryRoot();
+            string repoRoot = GetRepositoryRoot();
 
-            // Default for tests outside of the repo
-            var path = Path.Combine(Path.GetTempPath(), "NuGetTestFolder");
+            DirectoryInfo root;
 
-            if (repoRoot != null)
+            if (repoRoot == null)
             {
-                path = Path.Combine(repoRoot, ".test");
-                Directory.CreateDirectory(path);
-                path = Path.Combine(path, "work");
+                // Use a folder under %TEMP% if the repository root can't be determined
+                root = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), "NuGetTestFolder"));
             }
+            else
+            {
+                DirectoryInfo testDirectory = Directory.CreateDirectory(Path.Combine(repoRoot, ".test"));
 
-            Directory.CreateDirectory(path);
-            return path;
+                try
+                {
+                    File.WriteAllText(Path.Combine(testDirectory.FullName, "Directory.Build.props"), "<Project />");
+                    File.WriteAllText(Path.Combine(testDirectory.FullName, "Directory.Build.targets"), "<Project />");
+                    File.WriteAllText(Path.Combine(testDirectory.FullName, "Directory.Build.rsp"), string.Empty);
```

Comment:
How about committing these files to the repo `.test` directory?  The directory is in `.gitignore`, so some extra effort needs to be used to convince git to stage the files for commit. Or, edit `.gitignore` to ignore the directory, listing these 3 files as an exception to not ignore.

Reply:
I'm all for it, although I'd like to rename the folder to something else because my Windows console tab autocompletion always goes to `.test` first and `test` second which drives me nuts...  What do you think of:
1. `.testoutput`
2. `.testscratch`
3. `.testtemp`
4. Other?

Reply:
I can't reproduce the behaviour in cmd.exe or powershell or pwsh 🤷 . `cd t` then pressing tab cycles between `cd test` and `cd tools-local`, but never `cd .test`, in any 3 of those shells.  Although for powershell/pwsh, my `$PROFILE` has `Set-PSReadLineKeyHandler -Key Tab -Function Complete` because I hate the default behaviour.  Personally, `build.cmd` existing annoys me greatly, because it comes up in tab completion before `build.ps1` does, and my muscle memory still hasn't adapted.

Anyway, if we're really going to rename it, I guess `.testtemp` would be my preference, but since I can't reproduce what you're experiencing, I don't know if that will really solve the problem, or if the name needs to start with something other than `test`.

Reply:
Another way of solving this would be to set a property in the src/test folders that indicates whether the DBP in the root should be imported at all.

Reply:
I think for now I'm going to commit this and we can figure out if we want to just add the directory to the repo in the future.